### PR TITLE
Refactor/am 3258 iii

### PR DESCRIPTION
**Pull request by mstasgravitee**
## :id: Reference related issue. 
am 3258

## :man: :woman:@mentions of the person or team responsible for reviewing proposed changes

This is a nice example: https://github.com/gravitee-io/gravitee-access-management/pull/1822

### CODE REVIEW

File: gravitee-am-management-api/gravitee-am-management-api-rest/src/test/java/io/gravitee/am/management/handlers/management/api/resources/UserRolesResourceTest.java [hunk 1]
```diff
@@ -99,7 +99,7 @@ public void shouldGetUserRoles_emptyDynamic() {
         final Response response = getResponse(domainId, mockUser.getId(), true);
 
         assertEquals(HttpStatusCode.OK_200, response.getStatus());
-        assertEquals(response.readEntity(roles.getClass()).size(), 0);
+        assertEquals(0, response.readEntity(roles.getClass()).size());
```
Comment (podlesrafal) [inline gravitee-am-management-api/gravitee-am-management-api-rest/src/test/java/io/gravitee/am/management/handlers/management/api/resources/UserRolesResourceTest.java:102]:
```suggestion
        assertTrue(response.readEntity(roles.getClass()).isEmpty());
```

File: gravitee-am-management-api/gravitee-am-management-api-rest/src/test/java/io/gravitee/am/management/handlers/management/api/resources/UserRolesResourceTest.java [hunk 2]
```diff
@@ -111,7 +111,7 @@ public void shouldGetUserRoles_emptyRoles() {
         final Response response = getResponse(domainId, mockUser.getId(), false);
 
         assertEquals(HttpStatusCode.OK_200, response.getStatus());
-        assertEquals(response.readEntity(roles.getClass()).size(), 0);
+        assertEquals(0, response.readEntity(roles.getClass()).size());
```
Comment (podlesrafal) [inline gravitee-am-management-api/gravitee-am-management-api-rest/src/test/java/io/gravitee/am/management/handlers/management/api/resources/UserRolesResourceTest.java:114]: same for all assertions to 0 in this class

File: gravitee-am-reporter/gravitee-am-reporter-api/src/main/java/io/gravitee/am/reporter/api/provider/NoOpReporter.java [hunk 3]
```diff
@@ -99,4 +98,10 @@ public Reporter start() throws Exception {
     public Reporter stop() throws Exception {
         return this;
     }
+
+    private static void notBootstrappedLog() {
+        if (log.isDebugEnabled()) {
```
Comment (podlesrafal) [inline gravitee-am-reporter/gravitee-am-reporter-api/src/main/java/io/gravitee/am/reporter/api/provider/NoOpReporter.java:103]: why you check if is debugEnabled?

File: gravitee-am-management-api/gravitee-am-management-api-rest/src/test/java/io/gravitee/am/management/handlers/management/api/resources/UserRolesResourceTest.java (hunk 1 above)
Comment (mstasgravitee) [reply gravitee-am-management-api/gravitee-am-management-api-rest/src/test/java/io/gravitee/am/management/handlers/management/api/resources/UserRolesResourceTest.java:102]: fixed

File: gravitee-am-reporter/gravitee-am-reporter-api/src/main/java/io/gravitee/am/reporter/api/provider/NoOpReporter.java (hunk 3 above)
Comment (mstasgravitee) [reply gravitee-am-reporter/gravitee-am-reporter-api/src/main/java/io/gravitee/am/reporter/api/provider/NoOpReporter.java:103]: fixed

**Review by podlesrafal — APPROVED** [review]

**Pull request merged**